google/cloud/storage/internal/object_requests.h

```diff
@@ -473,9 +473,12 @@ struct ResumableUploadResponse {
   static StatusOr<ResumableUploadResponse> FromHttpResponse(
       HttpResponse&& response);
 
+  bool operator==(const ResumableUploadResponse& other) const;
+
   std::string upload_session_url;
   std::uint64_t last_committed_byte;
   std::string payload;
+  bool done;
```

Comment:
Per the GCSG, I would either change this to not be a bool *or* add a comment. For example:

```C++
ResumableUploadResponse{"", quantum - 1, {}, /*done=*/false}
```